packages/api/src/schema/api-schema.yaml

```
@@ -1981,7 +2273,12 @@ paths:
         schema:
           type: string
     get:
-      description: Gets a specific multistream target details
+      tags:
+        - Multistream
+      summary: Retrieve a multistream target
+      description: >-
+        You can retrieve a multistream target with its ID on
+        /multistream/target{id}.
```

This feels redundant with the OpenAPI spec itself. Would remove the path since it already shows up on the redoc ui
```suggestion
        You can retrieve a multistream target from its ID with this API.
```